### PR TITLE
Faq brench

### PR DESCRIPTION
**Pull request by kostia-hontarau**

### CODE REVIEW

File: wp-content/themes/target99/lib/less/components/about.less [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+    padding-bottom: 50px;
+
+    text-align: center;
+
+    @media @small-screen {
+      text-align: center;
+    }
+  }
+
+  .about-us__center {
```
Comment (kostia-hontarau) [inline wp-content/themes/target99/lib/less/components/about.less]: seems like a modificator

File: wp-content/themes/target99/lib/less/components/about.less [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+    display: inline-block;
+    min-width: 210px;
+    padding-bottom: 12px;
+
+    font-family: @main-control-font;
+    font-size: 24px;
+
+    text-transform: uppercase;
+
+    .about-us__title-module {
```
Comment (kostia-hontarau) [inline wp-content/themes/target99/lib/less/components/about.less]: Elements should not be nested, 
b -> e -> m

File: wp-content/themes/target99/lib/less/components/faq-ask-question.less [hunk 3]
```diff
@@ -0,0 +1,164 @@
+@import '../base/variables';
+@import '../base/layout';
+
+.faq-ask-question {
```
Comment (kostia-hontarau) [inline wp-content/themes/target99/lib/less/components/faq-ask-question.less]: faq will be enough

File: wp-content/themes/target99/lib/less/components/faq-ask-question.less [hunk 4]
```diff
@@ -0,0 +1,164 @@
...
+    }
+  }
+
+  .faq-ask-question__forms{
+    align-content: center;
+    display: table;
+    margin: 0 auto;
+    clear: both;
+
+    .right{
```
Comment (kostia-hontarau) [inline wp-content/themes/target99/lib/less/components/faq-ask-question.less]: modificator

File: wp-content/themes/target99/lib/less/components/faq-ask-question.less [hunk 5]
```diff
@@ -0,0 +1,164 @@
...
+        float: none;
+      }
+    }
+
+    @media @small-screen {
+      width: 100%;
+    }
+  }
+
+  .faq-ask-question__sm__input {
```
Comment (kostia-hontarau) [inline wp-content/themes/target99/lib/less/components/faq-ask-question.less]: Strange stuff

File: wp-content/themes/target99/lib/less/components/faq-info.less [hunk 6]
```diff
@@ -0,0 +1,49 @@
...
+
+    text-transform: uppercase;
+
+    color: @text-color-3;
+    border-bottom: 1px solid @text-color-1;
+  }
+
+}
+
+#accordions-76 {
```
Comment (kostia-hontarau) [inline wp-content/themes/target99/lib/less/components/faq-info.less]: Need to review this stuff together.

File: wp-content/themes/target99/templates/template-about.php [hunk 7]
```diff
@@ -1,48 +1,86 @@
...
-				if ( $the_query->have_posts() ) {
-					while ( $the_query->have_posts() ) {
-						$the_query->the_post();
-						$post_image_url = wp_get_attachment_image_src( get_post_thumbnail_id(), 'medium')[0];
-						$content_width_class = (bool)$post_image_url ? '' : 'news-post-single__details--full';
+    <section class="about-us">
+        <div class="about-us__inner-container layout__service">
+            <div class="about-us__title-container">
+                <div class="about-us__title">
+                    <h2 class="about-us__title-module">Движение "Цель 99"</h2>
```
Comment (kostia-hontarau) [inline wp-content/themes/target99/templates/template-about.php]: nested stuff

File: wp-content/themes/target99/templates/template-about.php [hunk 8]
```diff
@@ -1,48 +1,86 @@
...
+                </div>
+            </div>
+            <div class="about-us__text-answer">
+                Мы – это вы.<br>
+                Да-да. Не партия, не фракция, не государственная структура.<br>
+                Мы – это гражданское движение, к которому можете присоединиться вы, ваши родные, друзья, коллеги, соседи.<br>
+                Мы –это неравнодушные вы – те жители Беларуси, кто хочет и готов сделать свой дом чистым, двор – уютным, страну – ухоженной.
+            </div>
+
+            <div class="about-us__center">
```
Comment (kostia-hontarau) [inline wp-content/themes/target99/templates/template-about.php]: need to review this code together

**Review by kostia-hontarau — COMMENTED** [review]
Need to improve the code.

File: wp-content/themes/target99/lib/less/components/faq-ask-question.less (hunk 5 above)
Comment (rkarabanov) [reply wp-content/themes/target99/lib/less/components/faq-ask-question.less]: full half modificator

File: wp-content/themes/target99/lib/less/components/faq-info.less (hunk 6 above)
Comment (rkarabanov) [reply wp-content/themes/target99/lib/less/components/faq-info.less]: ++

File: wp-content/themes/target99/lib/less/components/faq-ask-question.less (hunk 3 above)
Comment (rkarabanov) [reply wp-content/themes/target99/lib/less/components/faq-ask-question.less]: feedback-form
faq change info

File: wp-content/themes/target99/templates/template-about.php (hunk 8 above)
Comment (rkarabanov) [reply wp-content/themes/target99/templates/template-about.php]: ++

File: wp-content/themes/target99/lib/less/components/faq-ask-question.less (hunk 5 above)
Comment (rkarabanov) [reply wp-content/themes/target99/lib/less/components/faq-ask-question.less]: ++

File: wp-content/themes/target99/lib/less/components/faq-ask-question.less (hunk 3 above)
Comment (rkarabanov) [reply wp-content/themes/target99/lib/less/components/faq-ask-question.less]: ++

File: wp-content/themes/target99/lib/less/components/faq-ask-question.less (hunk 4 above)
Comment (rkarabanov) [reply wp-content/themes/target99/lib/less/components/faq-ask-question.less]: ++

File: wp-content/themes/target99/templates/template-about.php (hunk 7 above)
Comment (rkarabanov) [reply wp-content/themes/target99/templates/template-about.php]: +-

File: wp-content/themes/target99/lib/less/components/about.less (hunk 2 above)
Comment (rkarabanov) [reply wp-content/themes/target99/lib/less/components/about.less]: ++

File: wp-content/themes/target99/lib/less/components/about.less (hunk 1 above)
Comment (rkarabanov) [reply wp-content/themes/target99/lib/less/components/about.less]: ++

**Pull request merged**